### PR TITLE
Add Bombermaaan

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/bjaraujo/Bombermaaan/issues/48
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/bjaraujo/Bombermaaan/pull/47

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.github.bjaraujo.Bombermaaan

File: com.github.bjaraujo.Bombermaaan.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/SDL/SDL_mixer-1.2.12.json
```
Comment (barthalion) [inline com.github.bjaraujo.Bombermaaan.yml:16]: Please check if sdl12-compat works.

Comment (Mailaender) [reply com.github.bjaraujo.Bombermaaan.yml:16]: I get this error https://github.com/bjaraujo/Bombermaaan/issues/46 at compile time even though I added `shared-modules/glu/glu-9.json` above it.

**Pull request closed without merging**